tools/checkdocs/checkdocs.go

```diff
@@ -58,6 +60,16 @@ func main() {
 				log.Print(err)
 				pass = false
 			}
+
+			if err = verifyDateNotPresent(fo); err != nil {
```

Comment:
I belive that will not work there because `fo` is already read until the end. I think `fo.Seek(…)` should do the trick.